src/MovieInfo/MovieInfo.jsx

```diff
@@ -0,0 +1,34 @@
+import "./style.css";
+import { movies } from "../Main/info";
+import Button from "../Button/Button";
+
+export default function MovieInfo({ movieId, onCloseMovieInfo }) {
```

Comment:
I think it's better to pass movie object to movie info because in the future we won't have global movies mock, and this data will be received from API.

Reply:
Changed the movie state in the App Component to pass a movie object instead of a movie id.